discover/deployment/articles-dxp/03-configuring-liferay/04-securing-liferay/01-saml/03-configuring-saml.markdown

```diff
@@ -0,0 +1,53 @@
+# Configuring SAML 
+
+As noted in the previous tutorials, anything related to configuring SP
+connections must be done through the SAML Admin UI where configurations are
+saved to Liferay's database. SP connections can no longer be made via properties
+files as they were in the past. 
+
+This is an instance-scoped configuration which can be managed via OSGi
+Configuration Admin. The affected properties are those in the
+`SAMLProviderConfiguration` metatype:
+
+    - `saml.keystore.credential.password`
+    - `saml.sp.assertion.signature.required`
+    - `saml.idp.authn.request.signature.required`
+    - `saml.sp.clock.skew`
+    - `saml.default.assertion.lifetime`
+    - `saml.sp.default.idp.entity.id`
+	- `saml.enabled`
+	- `saml.entity.id`
+	- `saml.sp.ldap.import.enabled`
+	- `saml.role`
+	- `saml.idp.session.maximum.age`
+    - `saml.idp.session.timeout`
+    - `saml.sp.sign.authn.request`
+    - `saml.sign.metadata`
+    - `saml.ssl.required`
+    - `saml.idp.metadata.name.id.attribute`
+
+The SAML Admin UI remains the place for creating the company-scoped
+configuration instances.
+
+Note that there is also a system wide configuration, represented by the
+`SamlConfiguration` metatype. This uses the 
+[Configuration API](/develop/tutorials/-/knowledge_base/7-0/configuration)
```

Comment:
It's actually the other way around, SamlProviderConfiguration is the one which uses Configuration API in order to read configuration at a company scope. The system scope only SamlConfiguration is handled by Configuration Admin natively.

Reply:
Are you sure? This came directly from your text in the ticket: https://issues.liferay.com/browse/LRDOCS-3683